interfaces/builtin/docker_support.go

```diff
@@ -161,12 +161,167 @@ pivot_root,
 # use 'privileged-containers: true' to support --security-opts
 
 # defaults for docker-default
-change_profile unsafe /** -> docker-default,
+# unfortunately, the docker snap is currently (by design?) setup to have both 
+# the privileged and unprivileged variant of the docker-support interface 
+# connected which means we rules that are compatible to allow both transitioning
```

Comment:
did you mean "which means we *have* rules"?

Reply:
yes good catch